testing/tiptop/APKBUILD

```diff
@@ -0,0 +1,28 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
```

Comment:
Can you remove this? We'd prefer to not have editor-specific things in APKBUILDs.